packages/fe-mockserver-core/test/unit/v2/__snapshots__/dataAccess.test.ts.snap

```diff
@@ -10,7 +10,7 @@ exports[`Data Access v2metadata - it can DELETE data for an entity 2`] = `
tity":false,"IsActiveEntity":false,"__metadata":{"id":"undefined/SEPMRA_C_PD_ProductText(Product='',Language='',DraftUUID=guid'',IsActiveEntity=false)","uri":"undefined/SEPMRA_C_PD_ProductText(Product='',Language='',DraftUUID=guid'',IsActiveEntity=false)","type":"SEPMRA_PROD_MAN.SEPMRA_C_PD_ProductTextType"}},"DraftAdministrativeData":{"DraftUUID":"","DraftEntityType":"","CreationDateTime":"/Date()/","CreatedByUser":"","LastChangeDateTime":"/Date()/","LastChangedByUser":"","DraftAccessType":"","ProcessingStartDateTime":"/Date()/","InProcessByUser":"","DraftIsKeptByUser":false,"EnqueueStartDateTime":"/Date()/","DraftIsCreatedByMe":false,"DraftIsLastChangedByMe":false,"DraftIsProcessedByMe":false,"CreatedByUserDescription":"","LastChangedByUserDescription":"","InProcessByUserDescription":"","__metadata":{"id":"undefined/I_DraftAdministrativeData(guid'')","uri":"undefined/I_DraftAdministrativeData(guid'')","type":"SEPMRA_PROD_MAN.I_DraftAdministrativeDataType"}},"__metadata":{"id":"undefined/SEPMRA_C_PD_Product(ProductID=3,DraftUUID=guid'',IsActiveEntity=true)","uri":"undefined/SEPMRA_C_PD_Product(ProductID=3,DraftUUID=guid'',IsActiveEntity=true)","type":"SEPMRA_PROD_MAN.SEPMRA_C_PD_ProductType"}},{"ProductID":4,"Product":"Acme_Safe","DraftUUID":"","IsActiveEntity":true,"HasActiveEntity":true,"HasDraftEntity":false,"Copy_ac":false,"ProductForEdit":"","ProductCategory":"","ProductPictureURL":"","to_ProductTextInOriginalLang":{"Preparation_ac":false,"Validation_ac":false,"Description_fc":0,"LanguageForEdit_fc":0,"Delete_mc":false,"Product":"","Language":"","LanguageForEdit":"","Name":"","Description":"","DraftUUID":"","DraftEntityCreationDateTime":"/Date()/","DraftEntityLastChangeDateTime":"/Date()/","ParentDraftUUID":"","HasActiveEntity":false,"HasDraftEntity":false,"IsActiveEntity":false,"__metadata":{"id":"undefined/SEPMRA_C_PD_ProductText(Product='',Language='',DraftUUID=guid'',IsActiveEntity=false)","uri":"undefined/SEPMRA_C_PD_ProductText(Product='',Language='',DraftUUID=guid'',IsActiveEntity=false)","type":"SEPMRA_PROD_MAN.SEPMRA_C_PD_ProductTextType"}},"DraftAdministrativeData":{"DraftUUID":"","DraftEntityType":"","CreationDateTime":"/Date()/","CreatedByUser":"","LastChangeDateTime":"/Date()/","LastChangedByUser":"","DraftAccessType":"","ProcessingStartDateTime":"/Date()/","InProcessByUser":"","DraftIsKeptByUser":false,"EnqueueStartDateTime":"/Date()/","DraftIsCreatedByMe":false,"DraftIsLastChangedByMe":false,"DraftIsProcessedByMe":false,"CreatedByUserDescription":"","LastChangedByUserDescription":"","InProcessByUserDescription":"","__metadata":{"id":"undefined/I_DraftAdministrativeData(guid'')","uri":"undefined/I_DraftAdministrativeData(guid'')","type":"SEPMRA_PROD_MAN.I_DraftAdministrativeDataType"}},"__metadata":{"id":"undefined/SEPMRA_C_PD_Product(ProductID=4,DraftUUID=guid'',IsActiveEntity=true)","uri":"undefined/SEPMRA_C_PD_Product(ProductID=4,DraftUUID=guid'',IsActiveEntity=true)","type":"SEPMRA_PROD_MAN.SEPMRA_C_PD_ProductType"}}]}}"`;
```

Comment:
in this snapshot the `__metadata` is something like

```json
{
  "id":"undefined/SEPMRA_C_PD_Product(ProductID=1,DraftUUID=guid'',IsActiveEntity=true)",
  "uri":"undefined/SEPMRA_C_PD_Product(ProductID=1,DraftUUID=guid'',IsActiveEntity=true)",
  "type":"SEPMRA_PROD_MAN.SEPMRA_C_PD_ProductType"
}
```
should we worry about `id` and `uri` starting with `undefined/`?

Reply:
not really i'tts just a missing urlPath